common/src/main/java/com/wynntils/features/user/inventory/AbilityTreeScrollFeature.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.inventory;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.FeatureCategory;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.mc.event.MouseScrollEvent;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wynn.utils.ContainerUtils;
+import java.util.regex.Pattern;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@FeatureInfo(stability = FeatureInfo.Stability.STABLE, category = FeatureCategory.INVENTORY)
+public class AbilityTreeScrollFeature extends UserFeature {
+
+    @Config
+    private boolean invertScroll = false;
```

Comment:
```suggestion
    @Config
    public boolean invertScroll = false;
```

Also, move this to be after the private fields.